### PR TITLE
SDK-986 update infrastructure

### PR DESCRIPTION
**Pull request by sergeyspryker**
Ticket: https://spryker.atlassian.net/browse/SDK-986

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 ###> symfony/framework-bundle ###
 APP_ENV=dev
+APP_DEBUG=1
```
Comment (pavelmaksimov25) [inline .env:18]: how it is going to help us to fix the issue?

File: infrastructure/sdk.Dockerfile [hunk 2]
```diff
@@ -51,6 +51,6 @@ RUN --mount=type=cache,id=composer,sharing=locked,target=/home/spryker/.composer
 
 ENV APP_ENV=prod
 
-RUN bin/console cache:warmup
+RUN bin/console cache:clear --no-debug
```
Comment (vol4onok) [inline infrastructure/sdk.Dockerfile:54]: I guess `cache:warmup` includes `cache:clear` inside f. or it's fix for issues with cache to use debug mode?

**Review by vol4onok — APPROVED** [review]

File: infrastructure/sdk.Dockerfile (hunk 2 above)
Comment (sergeyspryker) [reply infrastructure/sdk.Dockerfile:54]: `warmup` is included in cache clear

Comment (sergeyspryker) [reply infrastructure/sdk.Dockerfile:54]: The cache:clear command clears and warms up the application cache for a given environment
  and debug mode:

**Review by pavelmaksimov25 — DISMISSED** [review]

**Comment by pavelmaksimov25** [comment]
Dev QA with MacOS required.

**Review by pavelmaksimov25 — APPROVED** [review]

**Review by Kyrylo-Meretskyi — APPROVED** [review]

**Pull request merged**